templates/zerver/integrations/jitsi.md

```diff
@@ -0,0 +1,16 @@
+Zulip integrates with [Jitsi Meet](https://jitsi.org/jitsi-meet/),
+by default for video chat support. It is fully-encrypted, 100% open
+source video conferencing solution. Jitsi Meet is supported by
+default so no extra configuration is required to use it.
+
+### Using Jitsi Meet
+
+1. **Set jitsi as Video chat provider in Organization settings**.
+
+2. [Create and embed meeting links](/help/start-a-call) in your messages.
```

Comment:
I would say "You can now create Jitsi meetings using Zulip's [call button](/help/start-a-call)." for this and all the other variants.

Reply:
Yeah made these changes for `Jitsi Meet` and all other providers.